src/lib/auth-client.ts

```diff
@@ -0,0 +1,11 @@
+import { createAuthClient } from "better-auth/react"
+export const authClient = createAuthClient({
+    baseURL: process.env.NEXT_PUBLIC_APP_URL,
```

Comment:
where does this come from? why not just do VERCEL_URL ?? "http://localhost:3000"